src/components/Hero.js

```diff
@@ -17,9 +17,7 @@ export function Hero() {
     <div className="subheader">
       <div className="subheaderTitle">
         <div className="subheaderTitleContainer">
-          <p>
-            Where hiring <br /> comes together.
-          </p>
+          <p>Where hiring comes together.</p>
           <img
```

Comment:
> Images: I'd like to understand what are the best practices with image sizes when doing integration. Should we keep the image dimensions as there are on the figma file? Another question regarding images; I read different things about responsive images. But what are the best practices for images?


Good practices depend from scenario to scenario. I don't have any good resource (article) that talks about it in a comprehensive way. I'd need a more specific question, so that I could give you a better answer.

Nevertheless, here's some generic guidelines:
- The image ratio itself should stay the same, regardless of the screen size. The image should not stretch (change its ratio), otherwise, it would look ugly.
- If the image is irrelevant, you might hide it on smaller screens to save space.
- Using `img { max-width: 100%; }` is a good global approach to ensure images don't grow wider than their original size (to avoid blury images)